mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
+    })
+    .catch(error => { 
```

Comment:
.catch(error => { 
This catch block is incomplete and lacks proper error handling. It should be completed to handle errors properly. Additionally, it's recommended to maintain consistent error handling throughout the codebase for code reliability and readability.